clients/src/main/java/org/apache/kafka/clients/NodeApiVersions.java

```diff
@@ -68,7 +68,7 @@ public static NodeApiVersions create() {
      */
     public static NodeApiVersions create(Collection<ApiVersion> overrides) {
         List<ApiVersion> apiVersions = new LinkedList<>(overrides);
-        for (ApiKeys apiKey : ApiKeys.zkBrokerApis()) {
+        for (ApiKeys apiKey : ApiKeys.clientApis()) {
```

Comment:
This method is used in too many places so we should be careful about this change, we'd better keep it unchanged in this PR, and do this change in another patch.